client/commands/seeds/export.go

```diff
@@ -40,5 +44,20 @@ func exportSeed(cmd *cobra.Command, args []string) error {
 	}
 
 	// now get the output file and write it
-	return seed.Write(path)
+	return writeSeed(seed, path)
+}
+
+func writeSeed(seed certifiers.Seed, path string) (err error) {
+	var f *os.File
+	f, err = os.Create(path)
+	if err == nil {
+		stream := json.NewEncoder(f)
+		err = stream.Encode(seed)
+		f.Close()
```

Comment:
Just a drive-by comment: should we return early enough?
IMHO, we could rewrite this to return early and make the code intent somewhat clearer so
```go
func writeSeed(seed certifiers.Seed, path string) error {
    f, err := os.Create(path)
    if err != nil {
       if os.IsExist(err) {
           return nil
       }
       return errors.WithStack(err)
    }
    defer f.Close()
    
    stream := json.NewEncoder(f)
    if err := stream.Encode(seed); err != nil {
        return errors.WithStack(err)
    }
    return nil
}
```

Also am not sure if os.Create is going to return an error matching os.IsExist since the docs for os.Create say if the file already exists, it will be truncated https://golang.org/pkg/os/#Create
<img width="887" alt="screen shot 2017-07-22 at 1 05 52 am" src="https://user-images.githubusercontent.com/4898263/28489194-f1eb36ac-6e79-11e7-9afe-45526eaf0ed6.png">
I think that `os.IsExist(err)` check would work if we were doing os.Mkdir for which it'd fail if the directory/file already exists.